frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -768,27 +776,24 @@ const ManagePolicyPage = ({
         {showInheritedPoliciesButton && showInheritedTable && (
           <div className={`${baseClass}__inherited-policies-table`}>
             {globalPoliciesError && <TableDataError />}
-            {!globalPoliciesError &&
-              (isFetchingGlobalPolicies ? (
```

Comment:
Root issue: can't use a spinner to replace the table level or else it will re-render (and thus unfocus) the search input field